### PR TITLE
csv-to-share example

### PR DESCRIPTION
**Pull request by brian-vendia**
Initial version of csv-to-share.  Includes some boilerplate for contributions, etc.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+3. Ensure local tests pass.
+4. Commit to your fork using clear commit messages.
+5. Send us a pull request, answering any default questions in the pull request interface.
+6. Stay involved in the conversation.
+
+GitHub provides additional document on [forking a repository](https://help.github.com/articles/fork-a-repo/) and [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
+
+## Finding Contributions to Work On
+
+Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels ((enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/vendia/examples/labels/help%20wanted) issues is a great place to start.
```
Comment (cory-vendia) [inline CONTRIBUTING.md:63]: Double open-paren ( on line 63\.

**Review by cory-vendia — APPROVED** [review]
LGTM but left one comment on formatting

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# Contributing Guidelines
```
Comment (brett-vendia) [inline CONTRIBUTING.md:1]: For what it's worth, I generally use Contributor Covenant and modify https://github.com/vendia/serverless-express/blob/mainline/CODE_OF_CONDUCT.md

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
-# examples
\ No newline at end of file
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://raw.githubusercontent.com/vendia/serverless-express/mainline/vendia-logo.png" alt="vendia logo" width="100px">
```
Comment (brett-vendia) [inline README.md:3]: Probably add the logo to this repo and link to that

**Review by rpgreen — APPROVED** [review]
Nice!

**Review by brett-vendia — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (brian-vendia) [reply CONTRIBUTING.md]: Good catch!

File: CONTRIBUTING.md (hunk 2 above)
Comment (brian-vendia) [reply CONTRIBUTING.md:1]: Good call - added in update.

File: README.md (hunk 3 above)
Comment (brian-vendia) [reply README.md]: 👍 just added

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -64,8 +64,14 @@ Looking at the existing issues is a great way to find something to contribute on
 
 ## Code of Conduct
 
-Vendia has adopted the [Kind Humans Only](https://vendia.net/kind-humans) policy.  Remember to be kind to other contributors.  Please contact info@vendia.net with any additional questions or comments.
```
Comment (brett-vendia) [inline CONTRIBUTING.md]: Could you reference our Kind Humans Policy in the new CODE_OF_CONDUCT?

**Pull request merged**